src/cli.rs

```diff
@@ -22,3 +29,39 @@ impl Rover {
         }
     }
 }
+
+impl Report for Rover {
+    fn serialize_command(&self) -> Result<String, SputnikError> {
+        Ok(serde_json::to_string(&self)?)
+    }
+
+    fn is_enabled(&self) -> bool {
+        option_env!("APOLLO_TELEMETRY_DISABLED").is_none()
+    }
+
+    fn endpoint(&self) -> Result<Url, SputnikError> {
+        if let Some(url) = option_env!("APOLLO_TELEMETRY_URL") {
+            Ok(url.parse()?)
+        } else if cfg!(debug_assertions) {
+            Ok("http://localhost:8000".parse()?)
+        } else {
+            Ok("https://install.apollographql.com/telemetry".parse()?)
+        }
+    }
+
+    fn tool_name(&self) -> String {
+        option_env!("CARGO_PKG_NAME").unwrap_or("rover").to_string()
+    }
+
+    fn version(&self) -> String {
+        option_env!("CARGO_PKG_VERSION")
+            .unwrap_or("0.0.0")
```

Comment:
not sure i agree with this default?

Reply:
yeah... wasnt sure about this one. think we should panic by default instead? or return an error?